docs/CONFIGURATION.md

```diff
@@ -0,0 +1,218 @@
+- **backend.image.pullPolicy**: Polityka pobierania obrazu dla backendu. Opcje: `Always`, `IfNotPresent`, `Never`.
+- **backend.image.tag**: Tag obrazu Docker używanego dla backendu. Domyślnie: wersja aplikacji w chart.
+- **backend.serviceAccount.create**: Czy utworzyć nowe konto serwisowe dla backendu.
+- **backend.serviceAccount.automount**: Czy automatycznie montować API credentials konta serwisowego.
+- **backend.serviceAccount.annotations**: Adnotacje do dodania do konta serwisowego.
+- **backend.serviceAccount.name**: Nazwa konta serwisowego do użycia. Jeśli nie ustawiono i `create` jest ustawione na true, nazwa jest generowana przy użyciu szablonu fullname.
+- **backend.service.type**: Typ usługi do utworzenia dla backendu. Opcje: `ClusterIP`, `NodePort`, `LoadBalancer`.
+- **backend.service.port**: Port, na którym usługa backendu będzie dostępna.
+- **backend.livenessProbe.httpGet.path**: Ścieżka HTTP używana w liveness probe.
+- **backend.livenessProbe.httpGet.port**: Port używany w liveness probe.
+- **backend.readinessProbe.httpGet.path**: Ścieżka HTTP używana w readiness probe.
+- **backend.readinessProbe.httpGet.port**: Port używany w readiness probe.
+- **backend.autoscaling.enabled**: Czy włączyć autoskalowanie dla backendu.
+- **backend.autoscaling.minReplicas**: Minimalna liczba replik dla backendu.
+- **backend.autoscaling.maxReplicas**: Maksymalna liczba replik dla backendu.
+- **backend.autoscaling.targetCPUUtilizationPercentage**: Docelowe zużycie CPU dla autoskalowania.
+- **backend.autoscaling.targetMemoryUtilizationPercentage**: Docelowe zużycie pamięci dla autoskalowania.
+
+## Konfiguracja Frontend
+
+- **frontend.replicaCount**: Liczba replik dla wdrożenia frontendu.
+- **frontend.image.repository**: Repozytorium obrazu Docker dla frontendu.
+- **frontend.image.pullPolicy**: Polityka pobierania obrazu dla frontendu. Opcje: `Always`, `IfNotPresent`, `Never`.
+- **frontend.image.tag**: Tag obrazu Docker używanego dla frontendu. Domyślnie: wersja aplikacji w chart.
+- **frontend.service.type**: Typ usługi do utworzenia dla frontendu. Opcje: `ClusterIP`, `NodePort`, `LoadBalancer`.
+- **frontend.service.port**: Port, na którym usługa frontendu będzie dostępna.
+- **frontend.autoscaling.enabled**: Czy włączyć autoskalowanie dla frontendu.
+- **frontend.autoscaling.minReplicas**: Minimalna liczba replik dla frontendu.
+- **frontend.autoscaling.maxReplicas**: Maksymalna liczba replik dla frontendu.
+- **frontend.autoscaling.targetCPUUtilizationPercentage**: Docelowe zużycie CPU dla autoskalowania.
+- **frontend.autoscaling.targetMemoryUtilizationPercentage**: Docelowe zużycie pamięci dla autoskalowania.
+
+## Konfiguracja Ingress
+
+- **ingress.enabled**: Czy włączyć ingress dla aplikacji.
+- **ingress.className**: Nazwa klasy kontrolera ingress do użycia.
+- **ingress.annotations**: Adnotacje do dodania do zasobu ingress.
+- **ingress.hosts**: Lista hostów dla zasobu ingress. Każdy host może mieć własny zestaw ścieżek.
```

Comment:
I think this could be expanded a bit more. For example, you could mention that frontend paths will redirect to the frontend service, and backend paths to the backend service. This may seem obvious, but it’s worth including in the documentation for clarity.

Reply:
I don't want to overdo it since, we were supposed to fix ingress anyway

Reply:
Maybe. I don't know what will be fixed.

Reply:
I've changed it a bit let me know if itsfine now